Code/Editor/EditorViewportWidget.cpp

```diff
@@ -1957,6 +1979,24 @@ void EditorViewportWidget::SetDefaultCamera()
     PostCameraSet();
 }
 
+
+//////////////////////////////////////////////////////////////////////////
```

Comment:
nit: Please remove the `////////` lines  (this is generally just seen as visual noise and not super helpful)